index.html

```diff
@@ -0,0 +1,160 @@
+          analytics, monitor network health, tighten security measures and more. </p>
+      </div>
+      <div class="program-items">
+        <img src="assets/program_icon_03 1.png" alt="Augmented Reality image" />
+        <h4>Augumented Reality</h4>
+        <p>
+          Learn AI algorithms can be used in real-time to detect and track objects, allowing AR and VR experiences to
+          adapt to the user's movements and actions
+        </p>
+      </div>
+      <div class="program-items">
+        <img src="assets/program_icon_04 1.png" alt="Promt engineering image" />
+        <h4>Prompt Engineering</h4>
+        <p>
+          Discover top prompt engineering jobs on our niche job board, tailored to connect innovative companies with
+          skilled professionals in the field.
+        </p>
+      </div>
+      <div class="program-items">
+        <img src="assets/program_icon_05 1.png" alt="generative AI image" />
+        <h4>Generative AI</h4>
+        <p>
+          Generative AI is a type of artificial intelligence technology that can produce various types of content,
+          including text, imagery, audio and synthetic data.
+        </p>
+      </div>
+
+    </div>
+
+    <button class="program-btn" type="button"><strong>Join AIFG India Summit 2023</strong></button>
+    <a href="#" class="click-to-see"><strong>SEE THE WHOLE PROGRAM</strong></a>
+
+  </section>
+  <!-- program section end -->
+  <!-- speakers start -->
+  <section class="program-speakers" id="program-speakers">
+    <h3 class="program-speakers-title"><strong>Featured Speakers</strong></h3>
+
+    <div class="program-speaker-part"></div>
+    <button type="button" class="program-speaker-btn">More <img src="assets/arrow_down.png" alt="arrow_down"></button>
+  </section>
+  <!-- speakers end -->
+  <section class="our-partner" id="partner"> 
+    <h3 class="our-partner-title">Partner</h3>
+     <div class="our-partner-underline" ></div>
+    <div class="our-partner-part">
+       <div class="our-partner-logo">
+       <img src="assets/py.png" alt="partner 1">
+       <img src="assets/unesco.png" alt="partner 2">
+       <img src="assets/bill.png" alt="partner 3">
+       <img src="assets/unicef.png" alt="partner 4">
+       <img src="assets/un.png" alt="partner 5">
+       <img src="assets/imf.png" alt="partner 6">  
+     </div>
+    </div>
+  </section>
+  <!--Footer section starts-->
+  <div class="footer" id="footer">
+    <div class="footer-section">
+        <img class="footer-image" src="assets/footer image.png" alt=" footer logo ">
+        <p class="footer-description">Through AI for Good, we have the opportunity to revolutionize fields, empower communities, and find innovative solutions to complex global challenges, ultimately contributing to a brighter future for humanity.
+        </br></br>
+        <strong>AIFG 2023 © All rights reserved.</strong>
+        </p>
+    </div>
+</div>
```

Comment:
Both for your index and about page, I suggest you kindly wrap the footer section in a `footer` tag as this is more semantic😉. also, to match the design, kindly do well to add the necessary spacings (margin and padding)

design
<img width="697" alt="image" src="https://github.com/the-faizmohammad/conferencePage/assets/64643995/cf0b5ce4-ce20-4864-a5dc-060730f8cd98">

Yours
<img width="950" alt="image" src="https://github.com/the-faizmohammad/conferencePage/assets/64643995/9c8eaa12-74b4-419b-9ee0-fd5bdc3f316c">

Kindly implement these changes in both your index and about page👍.